user_stories.md

```diff
@@ -1,5 +1,17 @@
 *The Marketing Research Team*
+As a member of the MR team, I want to find out which products are most likely to be sucessful.
+As a member of the MR team, I want the test to be as unbiased as possible so that the results are valid.
+As a member of the MR team, I want to extract as much data from this project as I can to imporve my analytical model.
+As a member of the MR team, I want a large user sample in order to provide a more complete data set.
 
 *The Developer*
+As a developer, I want to design a well thought-out solution to solve the problem domain.
+As a developer, I want to meet the requirements of the research team in order to create a satisfactory product.
+As a developer, I want to meet the requirements of the focus group in order to make attractive and user friendly design.
```

Comment:
Your "I want to" and "in order to" feel backwards here... "I want to build an attractive and user-friendly site in order to meet the requirements of the focus group" makes more sense to me?